python/ray/serve/deployment.py

```diff
@@ -183,11 +183,6 @@ def user_config(self) -> Any:
     @property
     def max_concurrent_queries(self) -> int:
         """[DEPRECATED] Max number of requests a replica can handle at once."""
-
-        logger.warning(
```

Comment:
Why are we dropping this warning? I feel it will cause confusions for the users if nothing is logged, since when the user is calling `max_concurrent_queries`, they are actually getting `max_ongoing_requests`😅

Reply:
It was logging warnings when it wasn't supposed to, found a workaround :D